webapp/components/description/dataSourceDescriptionAndComments.js

```diff
@@ -34,8 +34,7 @@ const CommentableReviewDescriptions = props => {
           title={i18n.t('description.dataSourcesTitle')}
           section={section}
           name={dataSources}
-          template={dataSourcesEditorTemplate(i18n)}
-          countryIso={countryIso}/>
+          template={dataSourcesEditorTemplate(i18n)} />
```

Comment:
- I would use useCountryIso hook
- I would use useSelector(ReviewState.getOpenThreadTarget)
- countryIso is not a required prop
- I would remove mapStateToProps

Reply:
Unused file